lunatrace/bsl/hasura/migrations/lunatrace/1667411530061_build_logs_cascade/up.sql

```diff
@@ -0,0 +1,6 @@
+ALTER TABLE public.build_log
+    DROP CONSTRAINT build_log_build_id_fkey;
+
+ALTER TABLE public.build_log
+    ADD FOREIGN KEY (build_id) REFERENCES builds
```

Comment:
doesn't this already reference builds? https://github.com/lunasec-io/lunasec/blob/master/lunatrace/bsl/hasura/migrations/lunatrace/1662408856453_create_build_log/up.sql#L20